cookbook-react/src/recipes/HOCs/Spinner/components/loading.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
could we add a folder `Loading` and turn this `index.tsx` , to follow the standard?